test-resources/msgpack_test_fixtures.json

```diff
@@ -0,0 +1,75 @@
+[
+  {
+    "Name": "json array",
+    "Data": ["a", "b", "c"],
+    "Encoding": "json",
+    "NumRepeat": 0,
+    "Type": "jsonArray",
+    "MsgPack": "g7Bjb25uZWN0aW9uU2VyaWFsAKhtZXNzYWdlc5GCpGRhdGGtWyJhIiwiYiIsImMiXahlbmNvZGluZ6lqc29uL2pzb26pbXNnU2VyaWFsAA=="
```

Comment:
We should document that this is the msgpack encoding of a ProtocolMessage, not just a Message

Reply:
Also, when I msgpack decode this, I get:

```json
{
  "connectionSerial": 0,
  "messages": [
    {
      "data": "[\"a\",\"b\",\"c\"]",
      "encoding": "json/json"
    }
  ],
  "msgSerial": 0
}
```

Should the encoding be `json`, not `json/json`?

Reply:
> We should document that this is the msgpack encoding of a ProtocolMessage, not just a Message

But where should that be documented? json does not alllow comments.

Reply:
> Also, when I msgpack decode this, I get:
> 
> ```json
> {
>   "connectionSerial": 0,
>   "messages": [
>     {
>       "data": "[\"a\",\"b\",\"c\"]",
>       "encoding": "json/json"
>     }
>   ],
>   "msgSerial": 0
> }
> ```
> 
> Should the encoding be `json`, not `json/json`?

well caught...
fixed in [da9c383](https://github.com/ably/ably-common/pull/181/commits/da9c383c56b3eafb14e9d5339199d2ccb238e9f9)

Reply:
> But where should that be documented? json does not alllow comments.

In the other fixture files, we put a comment in a key starting with an underscore (e.g. [here](https://github.com/ably/ably-common/blob/main/test-resources/messages-encoding.json#L2) and [here](https://github.com/ably/ably-common/blob/main/test-resources/test-app-setup.json#L2)), and we could put it in the [README](https://github.com/ably/ably-common/tree/main/test-resources#readme), and we could put it in the [spec](https://sdk.ably.com/builds/ably/specification/main/features/#RSL6a1).

Reply:
added to the README.md in 5725a60